src/components/views/close-contact.tsx

```diff
@@ -1,71 +1,69 @@
 import React, {FC} from 'react';
-import {StyleSheet, Text, Linking} from 'react-native';
-import {useTranslation} from 'react-i18next';
+import {StyleSheet, Text} from 'react-native';
+import {useNavigation} from '@react-navigation/native';
 import PushNotification from 'react-native-push-notification';
+import {useTranslation} from 'react-i18next';
 
-import {BubbleIcons} from 'assets/icons';
-import {Button} from 'components/atoms/button';
+import {useCallBackStatus} from 'hooks/call-back-status';
+
+import {Spacing} from 'components/atoms/layout';
 import {Card} from 'components/atoms/card';
 import {Markdown} from 'components/atoms/markdown';
 import {Scrollable} from 'components/templates/scrollable';
-import {Spacing} from 'components/atoms/layout';
-import {text} from 'theme';
-import {useApplication} from 'providers/context';
-import {useSettings} from 'providers/settings';
 
-export const CloseContact: FC<any> = ({route}) => {
-  const {t} = useTranslation();
-  const {callBackData} = useApplication();
-  const {exposedTodo} = useSettings();
+import {text} from 'theme';
+import {BubbleIcons} from 'assets/icons';
 
-  const todoList = exposedTodo;
+const markdownStyles = {
+  text: {
+    ...text.large,
+    flexWrap: 'wrap'
+  },
+  strong: {
+    ...text.largeBold
+  },
+  block: {
+    marginBottom: 32
+  }
+};
 
-  const type = route.params && route.params.info;
+export const CloseContact: FC<any> = () => {
+  const {t} = useTranslation();
+  const navigation = useNavigation();
+  const {callBackIsQueued} = useCallBackStatus();
 
   PushNotification.setApplicationIconBadgeNumber(0);
 
   return (
-    <Scrollable
-      heading={type ? t('closeContact:infoTitle') : t('closeContact:title')}>
-      <Text style={text.largeBold}>
-        {type ? t('closeContact:intro') : t('closeContact:alertintro')}
-      </Text>
-      <Spacing s={16} />
-      {callBackData && !type && (
+    <Scrollable heading={t('closeContact:title')}>
+      <Markdown markdownStyles={markdownStyles}>
+        {t('closeContact:intro')}
+      </Markdown>
+      {callBackIsQueued ? (
         <>
-          <Card icon={<BubbleIcons.PhoneCall width={56} height={56} />}>
-            <Text style={text.largeBlack}>{t('closeContact:callBack')}</Text>
-            <Text style={styles.notice}>{t('closeContact:callBackQueue')}</Text>
+          <Text style={text.largeBold}>
+            {t('closeContact:callQueued:title')}
```

Comment:
this is not defined in the resources

Reply:
There is not a design for it. I'll merge it like this to force it being updated